index.html

```diff
@@ -5860,107 +5860,71 @@ <h3>Vocabularies</h3>
 
         <p class="issue" title="(AT RISK) URL values might change during Candidate Recommendation">
 This section lists URL values that might change during the Candidate
-Recommendation phase based on migration of documents to the W3C Technical
-Reports namespace and implementer feedback that requires the referenced URLs to
-be modified.
+Recommendation phase based on migration of documents to time-stamped locations,
+migration of documents to the W3C Technical Reports namespace, and/or
+implementer feedback that requires the referenced URLs to be modified.
         </p>
 
         <p>
-Implementations MUST ensure that the following vocabulary URLs used in the base
-context ultimately resolve to the following files, which are normative:
+Implementations that depend on RDF vocabulary processing MUST ensure that the
+following vocabulary URLs used in the base context ultimately resolve to the
+following files, which are normative. Other semantically equivalent
+serializations of the vocabulary files MAY be utilized by implementations.
```

Comment:
this is good, but will obviously break the hash... the MAY here means we don't need to test it.

Reply:
Yes, put that language in there to head off the inevitable: "But I want the vocabulary expressed in TURTLE or NQUADS" requests. This is the best compromise I could think of, open to other suggestions